src/databricks/labs/remorph/snow/snowflake.py

```diff
@@ -490,3 +507,38 @@ def _json_column_op(self, this, path):
                 return self.expression(local_expression.Bracket, this=this, expressions=[path])
 
             return self.expression(exp.Bracket, this=this, expressions=[path])
+
+        def _parse_window(self, this: exp.Expression | None, alias: bool = False) -> exp.Expression | None:
+            window = super()._parse_window(this=this, alias=alias)
+            if window:
+                # Adding default window frame for the rank-related functions in snowflake
+                if (
```

Comment:
These two `if conditions` can be merged into one condition, isn't it ?
`window and other conditions`...

Reply:
Done